### PR TITLE
Update third-party-review.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Replaced Jamie Hurewitz with Tim Quock.

### CODE REVIEW

**Comment by it33** [comment]
@justinegeffen can we use Amy Nicol for now instead of Tim Quock for now? 

She's going to be interim internal point-of-contact on legal operations until we bring someone in. 

We're currently interviewing for this role. There's a PR in to update our legal handbook with a link to this internal doc that lays out the legal AORs: https://docs.google.com/document/d/14KzrhszTlaibtM4APezTS8CFb1vaHxRsnkV9ZFef-Uc/edit

**Comment by it33** [comment]
Also, could I ask @justinegeffen and @cwarnermm to have me as reviewer on any PR change to our processes involving legal?

**Comment by justinegeffen** [comment]
Thanks, @it33 and apologies for not adding you. Legal PRs will include you as a mandatory reviewer.

File: operations/legal/contracts/third-party-review.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you have a contract from a third party that needs review, you can go to the [
 4. Select a **“Contract Type”**
 5. Choose the **“Counterparty”**. If the Counterparty is not in the list, select **“Invite a New Counterparty”** and fill out all of the required fields. Select **“Save”**.
 6. Select **“Request Review”**.
-7. Assign the Review to **“Jamie Hurewitz”**. 
+7. Assign the Review to **“Tim Quock”**.
```
Comment (justinegeffen) [inline operations/legal/contracts/third-party-review.md]:
```suggestion
7. Assign the Review to **“Amy Nicol”**. 
```

**Comment by justinegeffen** [comment]
@it33, are you open to me merging this? Thanks!

**Review by it33 — APPROVED** [review]

**Pull request merged**